docs-ui/components/code.tsx

```diff
@@ -64,7 +70,7 @@ const Code = ({children, className, label}: Props) => {
         {label && <Label>{label.replaceAll('_', ' ')}</Label>}
       </LabelWrap>
       <HighlightedCode className={className} ref={codeRef}>
-        <span>{children}</span>
+        {children}
```

Comment:
Contents won't update and highlight properly unless `children` is a direct child of `<HighlightedCode />`.